tests/test_entities.py

```diff
@@ -1990,7 +1990,7 @@ def test_generic(self):
                         response = method(**kwargs)
                 self.assertEqual(client_request.call_count, 1)
                 self.assertEqual(len(client_request.call_args[0]), 1)
-                self.assertEqual(client_request.call_args[1], kwargs)
+                assert set(kwargs).issubset(set(client_request.call_args[1]))
```

Comment:
I'm not sure it's a good idea to remove that strict equivalence check for all the entities as it can hide issues from us. Mind writing separate test for these 3 methods instead?